core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -27,12 +27,14 @@
           <div class="oppia-outer-container">
             <div class="oppia-user-and-rights">
               <div class="oppia-profile-details-container">
-                <span *ngIf="profilePictureDataUrl">
-                  <img [src]="profilePictureDataUrl"
+                <picture *ngIf="profilePicturePngDataUrl">
```

Comment:
Don't we also want to check for `profilePictureWebpDataUrl` in case that's falsey as well? Ditto below.

Reply:
Oh, I think we would not require that. The method to fetch both type of images are exactly same just differs in extension if the "png" is working fine so does "webp" and vice versa.
Thanks